xrm/index.d.ts

```diff
@@ -1677,16 +2145,12 @@ declare namespace Xrm
              *                               notes
              *                               timercontrol
              *                               kbsearch (CRM Online Only, use parature.d.ts)
+             *                               quickform (see ui.QuickForm)
+             *                               customcontrol: <namespace>.<name> (A custom control for mobile phone and tablet clients).
+             *                               customsubgrid: <namespace>.<name> (A custom dataset control for mobile phone and tablet clients).
              */
-            getControlType(): string;
-
-            /**
-             * Gets a boolean value, indicating whether the control is disabled.
-             *
-             * @return  true if it is disabled, otherwise false.
-             */
-            getDisabled(): boolean;
-
+            getControlType(): ControlType | string;
```

Comment:
Same here.  I know it is a breaking change, but it would be better to force the correct value imho